dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.netty-1.0.feature

```diff
@@ -0,0 +1,13 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.netty-1.0
+Subsystem-Name: Netty internal implementation 1.0
+singleton=true
+-features=\
+  io.openliberty.io.netty, \
+  io.openliberty.io.netty.ssl
+-bundles=\
+  io.openliberty.netty.internal, \
+  io.openliberty.netty.internal.impl
+kind=noship
+edition=core
+WLP-Activation-Type: parallel
```

Comment:
Just for clarification, the netty-1.0 private feature will eventually be used when we're ready to GA all of this code, and the SIP feature will be updated to leverage the private feature vs pulling in individual bundles? If that's the plan we should get an issue opened so we don't forget to clean this up.

Reply:
Issue opened, 20766.